### DISCUSSION TITLE
MSR now requires a phone number & location confirmation for each account

### DISCUSSION THREAD

**Discussion by udu3324** [charlesbel/Microsoft-Rewards-Farmer#321] [General]
This might be the end for the bot
The location information might be used to see if the number is in the same location, making free sms services useless (i think)

**Comment by EastArctica** [comment]
Just pay for a phone verification service, some allow local numbers. You're already profiting from the bot and usually these services don't cost very much

**Reply by udu3324** [reply]
Could you give me one?

**Reply by EastArctica** [reply]
https://sms-activation-service.com/ Charges $0.02 USD per account. However I have never used them and do not know if they won't just take your money and run.
